bin/codemods/run.js

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env node
+
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const path = require( 'path' );
+const child_process = require( 'child_process' );
+
+/**
+ * Internal dependencies
+ */
+const config = require( './src/config' );
+
+function getLocalCodemodFileNames() {
+	// Returns all JS files in bin/codemods/src folder, except for config and helpers.
+	const excludeUtilityFiles = name => name !== 'config.js' && name !== 'helpers.js';
+	const excludeDotFiles = name => name[0] !== '.';
+	const includeJSFiles = name => path.extname( name ) === '.js';
+	return fs.readdirSync( './bin/codemods/src' ).filter(
+		excludeUtilityFiles
+	).filter(
+		excludeDotFiles
+	).filter(
+		includeJSFiles
+	).map( name => name.replace( '.js', '' ) );
+
+}
+
+function getValidCodemodNames() {
+	return [
+		...getLocalCodemodFileNames(),
+		...Object.getOwnPropertyNames( config.codemodArgs ),
+	].map( name => '- ' + name ).sort();
+}
+
+function printableValidCodemodNames() {
+	return `${ getValidCodemodNames().join('\n') }`
```

Comment:
don't think the template literal is doing anything here. in fact, is this function even doing much? we'd save more actual code just calling `getValidCodemodNames().join( '\n' )`

Reply:
Whoops, fixed!